content/docs/start/experiments/experiment-versioning.md

```diff
@@ -0,0 +1,224 @@
+---
+title: 'Get Started: Experiment Versioning'
+description:
+  'Get started with versioning experiments with DVC. Track the changes to the
+  code, data, metrics, parameters and plots associated with each experiment,
+  without bloating your Git repo.'
+---
+
+# Get Started: Experiment Versioning
```

Comment:
I think the intro should at least mention that experiments are saved with git (I like to say "as hidden git commits"). Reasons:

1. Without this, some of the rest of the doc becomes a little confusing (like when we say to make sure you git add your code).
2. A big part of the value prop is that you manage both your code and experiment info in one place. We can't get into a list of benefits of our approach, but this one seems important enough to highlight.

Reply:
I think that, on the first page, we should focus on getting people to see stuff in the UIs as soon as possible, not distracting with other details, to match the effort to be onboarded in other tools. 

We do have some hints that something different is going on using the on-hover details that you improved and some mentions to Git. 
I think that should be all for this page

> A big part of the value prop is that you manage both your code and experiment info in one place. We can't get into a list of benefits of our approach, but this one seems important enough to highlight.

I think that is where the second page comes in. Quoting from your other comment:

> I'm not sure what to do with the experiment management page. It already feels like it goes pretty deep, certainly beyond where other experiment tracking tools go (since they don't really even cover this topic and some of it's not applicable to them). Should we be covering it in get started at all? 

Once people can see stuff in the UIs, I think it is good to have a separate page where we go into the value proposition against other tools. 
I agree we should try to keep it as lightweight as possible.

> Which commands/features are most important?

I think we need the 3 existing points `sharing`, `persisting`, and `cleaning`. Not covering one of those points would feel like an incomplete workflow to me.

Maybe we could try to reduce complexity by being more opinionated (i.e. don't mention `apply` but only `branch`)

Reply:
Okay, with the updated tooltip, it seems less problematic that we don't explain in the page itself.

> Once people can see stuff in the UIs, I think it is good to have a separate page where we go into the value proposition against other tools.

Do you think the 2nd page conveys that value prop?

Reply:
Okay, I think it's good enough for now. Resolving this one.